Helpers.js

```diff
@@ -0,0 +1,223 @@
+}
+
+
+
+/**
+ * Resets the Spreadsheet to default values
+ */
+function resetURLsToDefault(){
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Config").getRange("D2:I").deleteCells(SpreadsheetApp.Dimension.ROWS)
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Config").getRange("D2:H7").setValues(SpreadsheetApp.getActiveSpreadsheet().getSheetByName("web.dev Websites").getRange("A2:E7").getValues());
+}
+
+/**
+ * Delete all the data in Results tab
+ */
+function deleteData(){
+  // // Results
+  // let last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Results").getLastRow();
+  // SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Results").deleteRows(3,last_row+1-3)
+  // // Screenshots
+  // last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Screenshots").getLastRow();
+  // SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Screenshots").deleteRows(3,last_row+1-3)
+
+  let sheets = ["Results", "Screenshots", "Debug", "Accessibility", "Green Domains (GWF)", "Sustainability"]
+  for(let i = 0; i < sheets.length; i++){
+    let sheet_name = sheets[i]
+    let last_row = SpreadsheetApp.getActiveSpreadsheet().getSheetByName(sheet_name).getLastRow();
+    if(last_row >= 5){
+      SpreadsheetApp.getActiveSpreadsheet().getSheetByName(sheet_name).deleteRows(3,last_row+1-3)
+    }
+  }
+
+}
+
+/**
+ * Helper to understand the time it takes the Spreadsheet to flush
+ */
+function timePerformance() {
+  var startTime = new Date();
+  // Trigger a change in the sheet, for example, write to a cell
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName('Results').getRange('G2').setValue(Math.random());
+  SpreadsheetApp.flush(); // Apply all pending changes immediately
+  var endTime = new Date();
+  
+  var executionTime = endTime - startTime;
+  Logger.log("Execution Time: " + executionTime + " ms");
+  toast("Execution Time: " + executionTime + " ms");
+}
+
+
+
+/**
+ * Show an alert that we will call a 3P if the API is Green Domains
+ */
+function showAlertCallGWF() {
+  var ui = SpreadsheetApp.getUi(); // Same variations.
+  var result = ui.alert(
+     'Please confirm',
+     "Carbon's usage will be done through the use of services such as the Green Web Foundation’s Green Web Dataset. Those calls will send queries to The Green Web Foundation's dataset of green domains containing the information in this spreadsheet. Check https://developers.thegreenwebfoundation.org/api/greencheck/v3/check-single-domain/ for details. Do you still want to proceed?",
+      ui.ButtonSet.YES_NO);
+
+  // Process the user's response.
+  if (result == ui.Button.YES) {
+    // User clicked "Yes".
+    // ui.alert('Confirmation received.'); 
+    return false;
+  } else {
+    // User clicked "No" or X in the title bar.
+    ui.alert('Permission denied. Aborting'); 
+    return true;
+  }
+}
+
+
+
+/**
+ * Show an alert that we will call a 3P if the API is Green Domains
+ */
+function showAlertSustainability() {
```

Comment:
Same as for `showAlertCallGWF()`.

Reply:
done with `showConfirmationModalSustainability`